.github/workflows/prepare-for-minor-release.yml

```diff
@@ -1,5 +1,5 @@
 name: Prepare for minor release
-run-name: Update az-quickstart-scaffolding repository for new `${{ github.event.client_payload.release_branch_name }}` minor release branch.
+run-name: Update az-quickstart-scaffolding repository for new `${{ github.event.client_payload.release_branch_name }}` minor release branch (from ${{ github.event.client_payload.source_branch_name }}).
```

Comment:
```suggestion
run-name: Update az-quickstart-scaffolding repository for new `${{ github.event.client_payload.release_branch_name }}` minor release branch (from ${{ github.event.client_payload.dev_branch_source }}).
```

Reply:
`source_branch_name` is the actual name of the branch on the main repo that the workflow was triggered from

Reply:
Updated PR for other repo to include the new variable in the dispatch.  Thanks for noticing this.